das_api/src/builder.rs

```diff
@@ -27,7 +44,27 @@ impl RpcApiBuilder {
         module.register_alias("getAssetProof", "get_asset_proof")?;
 
         module.register_async_method("get_asset", |rpc_params, rpc_context| async move {
-            let payload = rpc_params.parse::<GetAsset>()?;
+            let payload: GetAsset;
+            if let Ok(parsed_payload) = rpc_params.parse::<GetAsset>() {
+                payload = parsed_payload;
+            } else {
+                let mut sequence_parser = rpc_params.sequence();
+
+                let id = match sequence_parser.next::<String>() {
+                    Ok(id) => id,
+                    Err(_) => {
+                        return Err(DasApiError::ValidationError(
+                            "'id' is missing or invalid".to_string(),
+                        )
+                        .into());
+                    }
+                };
+
+                payload = GetAsset {
+                    id,
+                    raw_data: sequence_parser.optional_next::<bool>().unwrap_or(None),
```

Comment:
I would prefer a nested field that looks like this: `"formatting": { "trimWhitespace": true/false }`

Reply:
And default `trimWhitespace to true`.